tests/integration/telemetry/tracing/tracing_test.go

```diff
@@ -0,0 +1,164 @@
+package tracing
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/bookinfo"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/ingress"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/zipkin"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/util/retry"
+	util "istio.io/istio/tests/integration/mixer"
+)
+
+var (
+	ist            istio.Instance
+	bookinfoNsInst namespace.Instance
+	galInst        galley.Instance
+	ingInst        ingress.Instance
+	zipkinInst     zipkin.Instance
+)
+
+func TestProxyTracing(t *testing.T) {
```

Comment:
non-blocking: Can we add a brief description of how the test functions here for ease in understanding?

Something like:

```
// TestProxyTracing exercises the trace generation features of Istio, based on the Envoy Trace driver for zipkin.
// The test verifies that all expected spans (one per each service in the sample bookinfo application) are generated
// and that they are all a part of the same distributed trace.
//
// More information on distributed tracing can be found here: ...
```

Reply:
Great suggestion and thanks for the wording! Done